supergiant/supergiant
# Unit tests fix

- fix app test
- disable 'should create' test because of uselessness
- add NO_ERRORS_SCHEMA for tests
- disable default 'should be created' because of no value
- update karma deps
- fix test config
